vowpalwabbit/memory.h

```diff
@@ -15,7 +18,11 @@ T* calloc_or_throw(size_t nmemb)
   { const char* msg = "internal error: memory allocation failed!\n";
     // use low-level function since we're already out of memory.
     fputs(msg, stderr);
+#ifdef VW_NOEXCEPT
+    return nullptr;
```

Comment:
What happens at the calling site when nullptr is returned?

Reply:
caller needs to check.

---
In reply to: [188081792](https://github.com/JohnLangford/vowpal_wabbit/pull/1481#discussion_r188081792) [](ancestors = 188081792)